ibm/service/power/resource_ibm_pi_cloud_connection_network_attach.go

```diff
@@ -36,19 +35,19 @@ func ResourceIBMPICloudConnectionNetworkAttach() *schema.Resource {
 
 		Schema: map[string]*schema.Schema{
 			// Required Attributes
-			helpers.PICloudInstanceId: {
+			Arg_CloudInstanceID: {
 				Type:        schema.TypeString,
 				Required:    true,
 				ForceNew:    true,
 				Description: "PI cloud instance ID",
 			},
-			helpers.PICloudConnectionId: {
+			Attr_CloudConnectionID: {
```

Comment:
helpers.PICloudConnectionId -> pi_cloud_connection_id
Attr_CloudConnectionID -> cloud_connection_id 
we can't redefine the variables in the schema.
Create a new variable called Arg_CloudConnectionID that has pi_cloud_connection_id as the value and put in the constants file.